### PR TITLE
Decode request body otherwise results in 415 error

### PR DESCRIPTION
**Pull request by zanguian**

### CODE REVIEW

File: web/api.go [hunk 1]
```diff
@@ -131,7 +132,7 @@ func main() {
 	server.POST("/get_access_token", func(c echo.Context) error {
 
 		req := new(Access)
-		err := c.Bind(req)
+		err := json.NewDecoder(c.Request().Body).Decode(&req)
```
Comment (rooftopsparrow) [inline web/api.go:135]: So the Bind() call doesn't work? I'd just followed this [guide](https://echo.labstack.com/guide/binding)

Comment (zanguian) [reply web/api.go:135]: It doesn't right now in Master. I can continue to debug in order to use Bind. Though there are many tickets on their repo related to it

Comment (rooftopsparrow) [reply web/api.go:135]: So is echo just a bad lib? None of the docs seem to work....

Do you wanna try to swap echo with gin?

Comment (zanguian) [reply web/api.go:135]: Not a bad idea, gin is the framework that Plaid endorsed. I'll look into it more tonight.

**Review by BProw — APPROVED** [review]

**Pull request merged**